lib/options.nix

```diff
@@ -1,11 +1,15 @@
 # Nixpkgs/NixOS option handling.
 { lib }:
 
-with lib.trivial;
-with lib.lists;
-with lib.attrsets;
-with lib.strings;
-
+let
+  inherit (lib)
+    isAttrs isBool isDerivation isFunction isInt isList isString
+    all collect concatMap concatLists elemAt filter foldl' head length mapAttrs optionals optional take
+    ;
```

Comment:
nit: the other import lists are each on their own line

Reply:
Done. Using merge-friendly syntax now.